tsung/templates/tsung-form-submission.xml.j2

```diff
@@ -47,8 +57,11 @@
 
 
             &save_form;
+            <thinktime value="2" random="true"></thinktime>
```

Comment:
yea, i guess i was hesitating since all of the examples used much higher values. maybe something like .5?